rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -297,16 +297,16 @@ fi
 
 # Test if compiler support hardening flags
 FILE=`mktemp`
-echo "void main() {}" > "$${FILE}.c"
-gcc -fPIE -pie -Wl,-zrelro -o "$${FILE}" "$${FILE}.c" 2>/dev/null
+echo "void main() {}" > "${FILE}.c"
+gcc -fPIE -pie -Wl,-zrelro -o "${FILE}" "${FILE}.c" 2>/dev/null
 if [ $? -eq 0 ]
 then
   SECURE_CFLAGS="-fPIE -pie"
   SECURE_LDFLAGS="-z relro"
 fi
-rm -f "$${FILE}.o" "$${FILE}" ))
+rm -f "${FILE}.o" "${FILE}"
 
-./configure --build=%_target --prefix=%{rudderdir} --with-workdir=%{ruddervardir}/cfengine-community --enable-static=yes --enable-shared=no %{openssl_arg} %{lmdb_arg} CFLAGS="${SECURE_CFLAGS}" LDFLAGS="${SECURE_LDFLAGS}"
+./configure --build=%_target --prefix=%{rudderdir} --with-workdir=%{ruddervardir}/cfengine-community --enable-static=yes --enable-shared=no %{openssl_arg} %{lmdb_arg} CFLAGS="${SECURE_CFLAGS} ${CFLAGS}" LDFLAGS="${SECURE_LDFLAGS}"
```

Comment:
I'd rather have CFLAGS at the beginning of the variable, so SECURE_CFLAGS would prevail over eventually conflicting options